sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -117,19 +118,54 @@ private object CosmosAccountConfig {
   }
 }
 
-private case class CosmosReadConfig(forceEventualConsistency: Boolean)
+private case class CosmosReadConfig(forceEventualConsistency: Boolean,
+                                    jsonSchemaConversionMode: JsonSchemaConversionMode)
+
+private object JsonSchemaConversionModes extends Enumeration {
+  type JsonSchemaConversionMode = Value
+
+  val Relaxed: JsonSchemaConversionModes.Value = Value("Relaxed")
+  val Strict: JsonSchemaConversionModes.Value = Value("Strict")
+}
 
 private object CosmosReadConfig {
+  private val DefaultJsonSchemaConversionMode: JsonSchemaConversionMode = JsonSchemaConversionModes.Relaxed
+
   private val ForceEventualConsistency = CosmosConfigEntry[Boolean](key = "spark.cosmos.read.forceEventualConsistency",
     mandatory = false,
     defaultValue = Some(true),
     parseFromStringFunction = value => value.toBoolean,
     helpMessage = "Makes the client use Eventual consistency for read operations")
 
+  private val JsonSchemaConversion = CosmosConfigEntry[JsonSchemaConversionMode](
+    key = "spark.cosmos.read.jsonSchemaConversionMode",
```

Comment:
`jsonSchemaConversionMode` why do we need json as prefix?

Cosmos DB is a json based. I would drop the `json` prefix unless if there is a specific reason for that?

Reply:
Renamed